test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -331,6 +460,11 @@ public void testAliasCreatedForCtorDepth1_1() {
     test("/** @constructor */ var a = function(){}; a.b = 1; "
          + "var c = a; c.b = 2; a.b != c.b;",
          "var a = function(){}; var a$b = 1; var c = a; c.b = 2; a$b != c.b;");
+
+    // Sometimes we want to prevent static members of a constructor from
+    // being collapsed.
+    testSame("/** @constructor */ var a = function(){};"
+        + "/** @nocollapse */ a.b = 1; var c = a; c.b = 2; a.b == c.b;");
```

Comment:
When we use nocollapse, would we want to prevent collapsing on all sub-properties of that property as well?